### DISCUSSION TITLE
firefox ignored github.githubassets.com (cross-origin problem)

### DISCUSSION THREAD

**Discussion by Linfiny-nagatani** [community/community#141526] [New to GitHub]
When using Firefox 131.0.3 to access a GitHub project page (https://github.com/username/projectname/), cross-origin constraints deny the request for https://github.githubassets.com/assets/*.js, preventing JavaScript-controlled actions such as the branch selection dropdown from working.

Although the site's list of supported browsers should include Firefox, access is not successful from Firefox (Chrome browser works fine).
(2024/10/15)

Do you have any information on this problem?
(The report at https://github.com/orgs/community/discussions/67058 may be due to the same factors.)

**Comment by ghostinhershell** [comment]
Hi @Linfiny-nagatani, thanks so much for reaching out in our community! 🚀👋🏾 

First I would recommend [clearing your cache and cookies](https://support.mozilla.org/en-US/kb/clear-cookies-and-site-data-firefox), just in case that ends up resolving this issue.

If that doesn't work, have you tried reaching out in the [Firefox Support Community](https://support.mozilla.org/en-US/questions/firefox) to see if this is a known issue on their end? When searching their community [I found this post](https://support.mozilla.org/en-US/questions/1444478) that seems like it may be siilar to what you're running into here.
